numpy/polynomial/tests/test_classes.py

```diff
@@ -116,7 +127,8 @@ def check_cast(Poly1, Poly2):
 #
 
 
-def check_identity(Poly):
+@pytest.mark.parametrize('Poly', classes, ids=classids)
```

Comment:
I think the "pytest way" to do this repeatedly is to define a "fixture" -- must admit I don't always find it as clear, but it is definitely less repetitive. Something like:
```
@pytest.fixture(params=classes, ids=classids)
def Poly(request):
    return request.param

def test_identity(Poly):
    ...
```

See https://docs.pytest.org/en/latest/fixture.html#fixture-parametrize

Reply:
Ah, I see what pytest has done: "One fixture to rule them all ..."